Client/mods/deathmatch/logic/luadefs/CLuaCameraDefs.h

```diff
@@ -31,6 +31,8 @@ class CLuaCameraDefs : public CLuaDefs
     static std::string                                                             GetCameraGoggleEffect();
     LUA_DECLARE(GetCameraFieldOfView);
     static unsigned char GetCameraDrunkLevel();
+    static CLuaMultiReturn<bool, float, float> GetCameraUnderwaterEffect();
+    static CLuaMultiReturn<bool, float>        GetCameraUnderwaterDarkness();
```

Comment:
Does `std::tuple` work here?

Reply:
After changing to `std::tuple`, the values will be returned as a table in the first argument. Should I make this change?

Reply:
no, thx